neo/Ledger/Blockchain.cs

```diff
@@ -364,7 +364,7 @@ private void OnNewHeaders(Header[] headers)
             system.TaskManager.Tell(new TaskManager.HeaderTaskCompleted(), Sender);
         }
 
-        private RelayResultReason OnNewTransaction(Transaction transaction)
+        private RelayResultReason OnNewTransaction(Transaction transaction, bool relay)
```

Comment:
Do we have a OnNewTransaction( tx, false ) somewhere? Should it default parameter to false?

Reply:
No need to set the default value.

Reply:
There is a `false` being used on the array of txs.